egs/wsj/s5/local/wsj_format_local_lms.sh

```diff
@@ -63,6 +206,32 @@ gunzip -c $lm_srcdir_3g/lm_unpruned.gz | \
 gunzip -c $lm_srcdir_4g/lm_unpruned.gz | \
   arpa2fst --disambig-symbol=#0 \
            --read-symbol-table=$lang/words.txt - data/lang${lang_suffix}_test_bd_fg/G.fst || exit 1;
+
+if [ $CLASSES == 1 ]; then
+  G_FST=data/lang${lang_suffix}_test_bd_fg/G.fst
+  G_WCLASS_FST=data/lang${lang_suffix}_test_bd_fg/G_wclass.fst
+  G_NO_WCLASS_FST=data/lang${lang_suffix}_test_bd_fg/G_no_wclass.fst
+  G_TMP_FST=data/lang${lang_suffix}_test_bd_fg/G_tmp.fst
+
+  cp $G_FST $G_WCLASS_FST
+
+  # go through all word classes in 'word_class_list.txt'
```

Comment:
It seems like this loop could have been broken out somehow as a script that you call... or at least there could have been more structure here.